src/Sulu/Bundle/SecurityBundle/Resources/public/js/models/permission.js

```diff
@@ -1,22 +0,0 @@
-/*
- * This file is part of the Sulu CMS.
- *
- * (c) MASSIVE ART WebServices GmbH
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-define(['mvc/relationalmodel'], function(relationalModel) {
-
-    'use strict';
-
-    return relationalModel({
-        defaults: function() {
-            return {
-                context: '',
-                permissions: ''
-            };
-        }
-    });
-});
```

Comment:
it was never used exept in the role model. but this relation has no advantage because we return them with the same rest api endpoint